app/scripts/migrations/092.ts

```diff
@@ -0,0 +1,49 @@
+import { cloneDeep } from 'lodash';
+import { hasProperty, isObject } from '@metamask/utils';
+
+export const version = 92;
+
+/**
+ * Add ticker to the providerConfig object if missing
+ *
+ * @param originalVersionedData - Versioned MetaMask extension state, exactly what we persist to dist.
+ * @param originalVersionedData.meta - State metadata.
+ * @param originalVersionedData.meta.version - The current state version.
+ * @param originalVersionedData.data - The persisted MetaMask state, keyed by controller.
+ * @returns Updated versioned MetaMask extension state.
+ */
+export async function migrate(originalVersionedData: {
+  meta: { version: number };
+  data: Record<string, unknown>;
+}) {
+  const versionedData = cloneDeep(originalVersionedData);
+  versionedData.meta.version = version;
+  versionedData.data = transformState(versionedData.data);
+  return versionedData;
+}
+
+function transformState(state: Record<string, unknown>) {
+  if (
+    hasProperty(state, 'NetworkController') &&
+    isObject(state.NetworkController) &&
+    hasProperty(state.NetworkController, 'providerConfig') &&
+    isObject(state.NetworkController.providerConfig)
+  ) {
+    const { providerConfig } = state.NetworkController;
+
+    if (providerConfig.ticker) {
+      return state;
+    }
+
+    state.NetworkController.providerConfig = {
+      ...providerConfig,
+      ticker: 'ETH',
+    };
```

Comment:
should the spread come last? 

```
    state.NetworkController.providerConfig = {
      ticker: 'ETH',
      ...providerConfig,
    };
```

Reply:
yes whoopsss

Reply:
Though if a ticker is present and truthy this won't even hit, but still I'll push the change.

Reply:
done here: [0f1f838](https://github.com/MetaMask/metamask-extension/pull/20367/commits/0f1f838c02b4d7b9e040643417a4deab4643d823)

Reply:
ahh i missed that guard somehow. I don't think the guard is needed if the spread comes last. No strong opinion though, feel free to leave as is